### PR TITLE
Add isDeviceSupported method

### PR DESCRIPTION
**Pull request by gtsonevv**
## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/near/near-api-js/blob/master/CONTRIBUTING.md).
- [x] Commit messages follow the [conventional commits](https://www.conventionalcommits.org/) spec
- [ ] **If this is a code change**: I have written unit tests.
- [x] **If this changes code in a published package**: I have run `pnpm changeset` to create a `changeset` JSON document appropriate for this change.
- [ ] **If this is a new API or substantial change**: the PR has an accompanying issue (closes #0000) and the maintainers have approved on my working plan.

### CODE REVIEW

File: packages/biometric-ed25519/src/index.ts
Comment (andy-haynes) [inline packages/biometric-ed25519/src/index.ts:146]: Is there any way to check for support without side effects?

I see that the current implementation uses the response from `createKey`, so it's probably not a straightforward thing. But if you are aware of a less intrusive check it would be helpful to document in a comment or follow-up issue.

If this is the necessary approach however, would it be better to use a different key each time? I.e. is the outcome the same if the `test-device` key doesn't exist vs was previously created?

File: packages/biometric-ed25519/src/index.ts
Comment (andy-haynes) [inline packages/biometric-ed25519/src/index.ts:143]: Can this comment please be updated to explain _what_ the device does or doesn't support? This is not immediately clear from the PR; as it stands this comment is not super descriptive 🙂

File: .github/workflows/pull-request.yml
Comment (andy-haynes) [inline .github/workflows/pull-request.yml:21]: If this is not relevant to this changeset please revert it from the workflows.

**Review by andy-haynes — CHANGES_REQUESTED** [review]
Looks good overall, some questions inline

File: packages/biometric-ed25519/src/index.ts
Comment (gtsonevv) [reply packages/biometric-ed25519/src/index.ts:146]: After spending some time researching how to do this we found that the only way to do it was by using the response from `createKey`. The outcome should be the same after each call.

Comment (andy-haynes) [reply packages/biometric-ed25519/src/index.ts:146]: Thank you for the clarification

**Comment by race-of-sloths** [comment]
@gtsonevv Thank you for your contribution! Your pull request is now a part of the [Race of Sloths](https://race-of-sloths.com)!
New Sloth joined the Race! Welcome!

[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://badge.race-of-sloths.com/gtsonevv?type=bot&theme=dark">
    <source media="(prefers-color-scheme: light)" srcset="https://badge.race-of-sloths.com/gtsonevv?type=bot&theme=light">
    Shows profile picture for the author of the PR
</picture>](https://race-of-sloths.com/profile/gtsonevv)


<details><summary>Current status: <i>waiting for scoring</i></summary>

We're waiting for maintainer to score this pull request with `@race-of-sloths score [0,1,2,3,5,8,13]` command
</details><details><summary>What is the Race of Sloths</summary>
<p>
Race of Sloths is a friendly competition where you can participate in challenges and compete with other open-source contributors within your normal workflow

For contributors:

- Tag @race-of-sloths inside your pull requests
- Wait for the maintainer to review and score your pull request
- Check out your position in the [Leaderboard](https://race-of-sloths.com/leaderboard)
- Keep weekly and monthly streaks to reach higher positions
- Boast your contributions with a dynamic picture of your [Profile](https://race-of-sloths.com/profile/gtsonevv)

For maintainers:
- Score pull requests that participate in the Race of Sloths
- Engage contributors with fair scoring and fast responses so they keep their streaks
- Promote the Race to the point where the Race starts promoting you
- Grow the community of your contributors

Feel free to check [our website](https://race-of-sloths.com) for additional details!
</p>
</details> 

<details>
<summary>Bot commands</summary>

- For contributors
  - **Include a PR:** `@race-of-sloths include` to enter the Race with your PR
- For maintainers:
  - **Assign points:** `@race-of-sloths score [1/2/3/5/8/13]` to award points based on your assessment.
  - **Reject this PR:** `@race-of-sloths exclude` to send this PR back to the drawing board.
  - **Exclude repo:** `@race-of-sloths pause` to stop bot activity in this repo until `@race-of-sloths unpause` command is called

</details>

**Comment by andy-haynes** [comment]
@race-of-sloths score 3

**Review by andy-haynes — APPROVED** [review]
🚢

**Comment by race-of-sloths** [comment]
### 🌟 Score recorded!

@andy-haynes, thank you for scoring this pull request in the Race of Sloths!

**Comment by race-of-sloths** [comment]
### ✅ PR is finalized! 
Your contribution is much appreciated with a final score of 3!
You have received 40 (30 base + 10 weekly bonus) Sloth points for this contribution


Another weekly streak completed, well done @gtsonevv! To keep your weekly streak and get another bonus make pull request next week! Looking forward to see you in race-of-sloths

**Pull request merged**
